t/bad_databases_t.c

```diff
@@ -1,5 +1,8 @@
 // This test currently does not work on Windows as nftw is
 // not available.
+#ifndef _POSIX_C_SOURCE
+#define _POSIX_C_SOURCE 200809L
+#endif
```

Comment:
It doesn't matter too much, but could we have just set these in `maxminddb_test_helper.h`?